main.rb

```diff
@@ -0,0 +1,123 @@
+require './app'
+
+class Main
+  attr_accessor :app, :list_of_books, :list_of_people
+
+  def initialize
+    @app = App.new
+    @list_of_books = []
+    @list_of_people = []
+  end
+
+  # list of options
+  def options
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
```

Comment:
- [ ] Good Job so far , but after you implemented the previous changes the app produced a new error when creating either a student or a teacher, kindly fix it 🙏 

teacher errors : 
![image](https://user-images.githubusercontent.com/89282221/235310513-aa50c580-b622-4f4a-9425-37e48241253c.png)


student errors : 
![image](https://user-images.githubusercontent.com/89282221/235310550-b02402a6-1bc1-4192-a376-2409860f1724.png)